task 26.py

```diff
@@ -0,0 +1,13 @@
+# '''Задача 26: Напишите программу, которая на вход принимает два числа A и B,
+# и возводит число А в целую степень B с помощью рекурсии.
+# A = 3; B = 5 -> 243 (3⁵)
+# A = 2; B = 3 -> 8'''
+
+def recApowB(a, b):
+    if b == 0:
+        return 1
```

Comment:
Можно сократить на один вызов функции.
```suggestion
    if b == 1:
        return a
```